src/include/config.h

```diff
@@ -0,0 +1,33 @@
+#ifndef IGRAPH_PRIVATE_CONFIG_H
+#define IGRAPH_PRIVATE_CONFIG_H
+
+#include "igraph_config.h"
+
+#define HAVE_STRCASECMP 1
+#define HAVE_STRNCASECMP 1
+/* #undef HAVE__STRICMP */
+/* #undef HAVE__STRNICMP */
+#define HAVE_STRDUP 1
+#define HAVE_STRNDUP 1
+// #define HAVE_USELOCALE 1
+/* #undef HAVE_XLOCALE */
+/* #undef HAVE__CONFIGTHREADLOCALE */
+
+#define HAVE_BUILTIN_OVERFLOW 1
+
+/* #undef HAVE__UMUL128 */
+/* #undef HAVE___UMULH */
+#define HAVE___UINT128_T 1
+
+#define HAVE_GLPK 1
+#define HAVE_LIBXML 1
+
+/* #undef INTERNAL_BLAS */
+/* #undef INTERNAL_LAPACK */
+/* #undef INTERNAL_ARPACK */
+/* #undef INTERNAL_GMP */
+
+#define IGRAPH_F77_SAVE static __thread
```

Comment:
This is only used for `f2c`-generated code which we don't use for R/igraph (we vendor the original Fortran ARPACK code instead).